GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -385,11 +401,16 @@ public virtual int ExitCode
                 get { return this.Process.ExitCode; }
             }
 
-            public virtual bool TryStart(string path, string args, out Exception exception)
+            public virtual void WaitForExit()
+            {
+                this.Process.WaitForExit();
+            }
+
+            public virtual bool TryStart(string path, string args, bool useShellExecute, out Exception exception)
             {
                 this.Process.StartInfo = new ProcessStartInfo(path)
                 {
-                    UseShellExecute = true,
+                    UseShellExecute = useShellExecute,
                     WorkingDirectory = Environment.SystemDirectory,
                     WindowStyle = ProcessWindowStyle.Normal,

```

Comment:
is this trying to change window style of the parent process's window, when running inline?

Reply:
No - It does not want to change the parent process's window style... `WindowStyle`'s default value is `Normal`.